app/controllers/carts_controller.rb

```diff
@@ -3,13 +3,16 @@ def index
    @cart_items = session[:cart]
     @ordered_foods = {}
     @total = 0
-    @cart_items.each do |food_id, qty|
+    @prep_total = 0
+    @cart_items.each do |food_id, qty, prep_time|
       food = Food.find(food_id)
-      @ordered_foods[food_id] = { :food => food , :qty => qty }
-      check_food_status(food, qty, food_id)
+      prep_time = line_prep_total(qty, food.prep_time) 
+      @ordered_foods[food_id] = { :food => food , :qty => qty, :prep_time => prep_time.to_i }
```

Comment:
Line is too long. [93/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space found before comma.